### ISSUE TITLE
 Request for auto color options like "only" "not" etc...?

### ISSUE THREAD

**Issue by Sound-Asleep** [reaper-oss/sws#1868]
Like the title says, options so that i can require exact phrase, so that words contained in other words may not trigger auto color, like "bass" might not want that if i'm recording "bassoon" or whatever. But, I might want clap to be claps clapper, clapping, handclap whatever. So, the option to select "and" "or" "not" "only" and I think that's it. Would be sweet. autocolor is already great, but this would make it perfect.

**Comment by mojosmoothmusic** [comment]
I would also suggest using double quotes to indicate exact matches only, for example:

"name" would override 'folder name'

**Comment by Buy-One** [comment]
Using asterisk * to indicate a part of the word would be more consistent because SWS extension already uses this syntax in track selection rules of ReaConsole.

So `TEST` is exact match, `TEST*` matches TESTS, TESTING etc., `*TEST` matches PROTEST, DETEST etc., `*TEST*` matches PROTEST, TESTING, PROTESTING etc.

**Comment by rellimneb** [comment]
yes this would be very helpfull, and the ability to have it case sensitive or not would also be usefull.
